serverlessworkflow/modules/ROOT/pages/tooling/serverless-logic-web-tools/serverless-logic-web-tools-openshift-integration.adoc

```diff
@@ -65,15 +39,11 @@ image::tooling/serverless-logic-web-tools/serverless-logic-web-tools-openshift-i
 . Enter your OpenShift project name in the *Namespace (project)* field.
 . Enter the value copied value of `--server` flag in the *Host* field.
 . Enter the value of API token in the *Token* field.
+. If your OpenShift instance uses Self-Signed certificates, then you must also check the Insecurely disable TLS certificate validation option.
```

Comment:
Is `Insecurely disable TLS certificate validation` a UI element? If yes, then please make it bold. 

```suggestion
. If your OpenShift instance uses Self-Signed certificates, you must also check the *Insecurely disable TLS certificate validation* option.
```